### ISSUE TITLE
Link with Popup in List Report

### ISSUE THREAD

**Issue by thomasmarkert** [oblomov-dev/abap2UI5#230]
Hi,

Is it possible to include a link with a popup in the "Z2UI5_CL_APP_DEMO_49" example?

SAP Example
![image](https://github.com/oblomov-dev/abap2UI5/assets/123086939/d56118b4-d98a-4994-86fb-d466bc6beb25)

Best Regards 
Thomas

**Comment by choper725** [comment]
its a popup or a popover.. it looks like a popover from the pic, can you link to the example the pic is from?

**Comment by oblomov-dev** [comment]
looks like a fiori elements floorplan and is a popover at a specific item position of a table, i will check this.

**Comment by oblomov-dev** [comment]
I added the demo z2ui5_cl_app_demo_52:
<img width="800" alt="image" src="https://github.com/oblomov-dev/abap2UI5/assets/102328295/d05715a0-0625-413a-a45a-e9baef1752ec">

You can change the popover as you want and use the controls you need in your use case.

Best regards.

**Comment by thomasmarkert** [comment]
As always, thank you for the great work. I looked at the example Z2UI5_CL_APP_DEMO_52. Is it possible to transfer the ID of the line. In the field lv_id there is a value with which I unfortunately can't do anything.

![image](https://github.com/oblomov-dev/abap2UI5/assets/123086939/dcef0d67-b188-4173-b6af-160f49d7a1ee)

Best Regards
Thomas

**Comment by choper725** [comment]
@oblomov-dev can explain what ‘ ${$source>/id}’ mean ?

**Comment by thomasmarkert** [comment]
would have expected the value e.g. {COUNT} to be returned here.

**Comment by oblomov-dev** [comment]
i needed to extend the _event method again. You can check demo z2ui5_cl_app_demo_52. 

Now you can import three parameters (data, data2, data3). With the first one we read the fontend ID to place the popover at the correct position and with the second one we read the count entry of the row:
<img width="1079" alt="image" src="https://github.com/oblomov-dev/abap2UI5/assets/102328295/9fea4389-1105-4d7f-8f89-c0aa8287ed84">
<img width="432" alt="image" src="https://github.com/oblomov-dev/abap2UI5/assets/102328295/049d7fd4-a344-49e2-8642-388c82414f18">

@choper725 
The object Source, Event and Parameters are "build in" object of UI5 we can use out-of-the-box in an XML-View.
${$source>/id} means use the attribute ID of the event source.
You can check this documentation for more infos:
https://openui5.hana.ondemand.com/#/topic/b0fb4de7364f4bcbb053a99aa645affe

Best regards

**Comment by thomasmarkert** [comment]
Thanks very much. As always, great support. All I can say is thank you for the great help.
Can you already use ABAP2UI5 productively?

**Comment by thomasmarkert** [comment]
Thanks very much. As always, great support. All I can say is thank you for the great help.
Can you already use ABAP2UI5 productively?

**Comment by oblomov-dev** [comment]
yes, you can transport abap2UI5 to the productive system as it is, i would recommend doing the following:
1. make sure to install the new abap2UI5 version (demos are separated now, there is no need to transport all the demos to the productive system)
2. transport this with your http/icf handler to the productive system
3. start the hello world app and check if it is working properly, if not very often the following issues occur:
         - check if the ICF service is activated (SICF)
         - check if the UI5 library is loaded
4. when hello world is running you know that your own developed apps will also technically work in the productive system and you can continue with transporting & trying out your other apps…

Basically abap2UI5 is just an http service and handler implementation and it does not side effect any other parts of your system, so there is nothing you have to be worry about. Just try it out and feel free to open an issue if you run into problems.
